docker/php-fpm/production.ini

```diff
@@ -0,0 +1,2 @@
+opcache.preload = /project/config/preload.php
+opcache.preload_user = root
```

Comment:
Acting as `root` should generally be avoided. But it isn't the main focus of this pull request.